### ISSUE TITLE
Trello node fails with 403 on GET operations

### ISSUE THREAD

**Issue by faevourite** [n8n-io/n8n#8713]
### Bug Description

All "get" operations like getting a board fail with a 403 error. It's likely due to [this deprecation notice](https://developer.atlassian.com/cloud/trello/changelog/#CHANGE-1459) on the Trello API dev changelog. It suggests that the Trello node is sending a payload with GET requests.

Also, see [this community post](https://community.n8n.io/t/get-operations-in-trello-node-fail/40670).

Part of the error:
```
ERROR: Forbidden - perhaps check your credentials?
403 - "\n\nERROR: The request could not be satisfied\n\n403 ERROR\nThe request could not be satisfied.\n\nBad request.\nWe can't connect to the server for this app or website at this time. There might be too much traffic or a configuration error. Try again later, or contact the app or website owner.\n
\nIf you provide content to customers through CloudFront, you can find steps to troubleshoot and help prevent this error by reviewing the CloudFront documentation.\n
\n\n\nGenerated by cloudfront (CloudFront)
```

![2024-02-22-6UiXWJeD](https://github.com/n8n-io/n8n/assets/387926/5d1a6142-f48d-4f14-9644-163ffba61e62)

### To Reproduce

1. Add a Trello node with a get, such as "Get Board"
2. Provide a board id
3. Test the step

### Expected behavior

Should succeed. Now returns a 403 matching the one from the linked deprecation notice.

### Operating System

Ubuntu via official Docker

### n8n Version

1.27.2

### Node.js Version

18.19.0

### Database

SQLite (default)

### Execution mode

main (default)

**Comment by Joffcom** [comment]
Hey @faevourite,

It looks like someone has popped in a PR to fix this one so as soon as the CLA is signed by the user we will be able to get it merged in for a future release.

**Comment by janober** [comment]
Fix got released with [`n8n@1.30.1`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.30.1)
